modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -312,7 +317,24 @@ export default {
       }
     }
   },
-  mounted() {
+  async mounted() {
+    // Fetch the user data of the person who last edited this doc for display in the admin bar
+    if (this.moduleOptions.context.updatedBy && this.moduleOptions.context.updatedBy !== 'ApostropheCMS') {
+      const editor = await apos.http.get(
+        `${window.apos.user.action}/${this.moduleOptions.context.updatedBy}`,
```

Comment:
Thinking ahead, this won't be possible for non-admin users to fetch. I suggest that updatedBy just be set to an object with all the properties you need for display at the time you set it. This could get out of date but it's a snapshot of what their name was at the time so that's probably OK and as long as you include `_id` among those properties we can better it later.